src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Grrrr...it begin to make me mad. This is a mess, as they not able to choose the correct format by themself.
I think it's because your build don't have `libnpp` included, it's why i withdrawed the others command lines given at first.

Nice graphic card, though i'm afraid that they will fry my super CPU :rofl: